CONTRIBUTING.md

```diff
@@ -1,154 +1,13 @@
-[#](#) Contributing to Syft
+# Contributing
 
-If you are looking to contribute to this project and want to open a GitHub pull request ("PR"), there are a few guidelines of what we are looking for in patches. Make sure you go through this document and ensure that your code proposal is aligned.
+Thank you for your interest in contributing to Syft!
 
-## Setting up your environment
+Please see the [contribution guide](https://oss.anchore.com/docs/contributing/syft/) for development requirements and helpful tips to get started developing in the repo. For a deeper dive, please see the [architecture docs](https://oss.anchore.com/docs/architecture/syft/).
 
-Before you can contribute to Syft, you need to configure your development environment.
+**Have a question or need help?** Check out our [issues and discussions guide](https://oss.anchore.com/docs/contributing/issues-and-discussions/) to find the right place to start a conversation.
 
-### Debian setup
+**Ready to submit code?** Our [pull request guide](https://oss.anchore.com/docs/contributing/pull-requests/) covers everything from title conventions to the review process. Don't forget that all commits require a [sign-off](https://oss.anchore.com/docs/contributing/sign-off/).
```

Comment:
Mention of the Sign-Off should probably be "big and bold" at the beginning so people don't miss it